homeassistant/components/http/auth.py

```diff
@@ -92,6 +93,7 @@ async def async_validate_signed_request(request):
             return False
 
         request[KEY_HASS_USER] = refresh_token.user
+        request[KEY_HASS_CREDENTIAL] = refresh_token.credential
```

Comment:
I'm not entirely happy with this. Mainly that we actually have a bit too much data in the credential. Should we perhaps only expose the credential_id here?

Reply:
I've reworked this to only expose the refresh token id used for the request (previously it exposed the credential object). This is less sensitive. You'd need to ask for the raw data from the auth manager.